src/App.tsx

```diff
@@ -1,61 +1,134 @@
 import './App.scss';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { Todos } from './types/Todos';
+import { User } from './types/User';
+
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const findUser = usersFromServer.find(user => user.id === userId);
+
+  return findUser || null;
+}
+
+export const todos: Todos[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [newTitle, setNewTitle] = useState('');
```

Comment:
Consider renaming the state variables `newUser` and `setNewUser` to `todos` and `setTodos` respectively, to better represent their purpose.